.circleci/config.yml

```diff
@@ -339,6 +352,9 @@ workflows:
       - static_code_analysis_with_detekt:
           requires:
             - build
+      - code_linting_check_with_ktlint:
```

Comment:
only minor suggestion would be run the auto-correction task before code analysis. I mean the order of execution would be 
1.  Build
2. ./gradlew ktlintCheck
3. ./gradlew detekt
4. Unit test
5. Integration test

Reply:
Makes sense, have done this.

Reply:
@sudhakar-plr this isn't an autocorrection task though, if linting fails this pipeline stage will fail.  It will be down to the developers to check their branches using`./gradlew ktlintCheck` locally, and then running `./gradlew ktlintFormat` to fix any issues (ideally) before pushing.

I've added a couple of lines to the README to explain this.